### DISCUSSION TITLE
ActivityPods and NextGraph are joining forces

### DISCUSSION THREAD

**Discussion by giscus** [assemblee-virtuelle/activitypods#355] [Blog articles]
# ActivityPods and NextGraph are joining forces

Both projects are based on Semantic Web/Linked Data/RDF technologies, and both foster the decoupling of data storage from data usage. Data storage should stay in total control of the end-user, and third party apps should request permission to access or modify such data.

https://activitypods.org/activitypods-and-nextgraph-are-joining-forces

**Comment by prosandmusic** [comment]
Great work! Currently experimenting with bringing ActivityPods to production.

**Reply by Laurin-W** [reply]
Cool to hear :)
If you run into issues or have feedback, feel free to share!
